src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -830,31 +828,6 @@ class Simpletask : ThemedActivity() {
             return saved_filters
         }
 
-    /**
-     * Handle filter import and export
-     */
-
```

Comment:
App structure/layout stuff should really be in XML if possible. Turns out it is possible (see xml files changed).